keyboards/redox/keymaps/jeherve/rules.mk

```diff
@@ -0,0 +1,7 @@
+RGBLIGHT_ENABLE = yes
+UNICODE_ENABLE = yes
+AUDIO_ENABLE = yes
+
+ifndef QUANTUM_DIR
```

Comment:
This block should be removed.

It's a holdover from the old makefile system, and no longer used.